src/styles/blocks/cards.scss

```
@@ -0,0 +1,116 @@
+.cards-catalog {
+  margin: 0 auto;
+  width: getWidthCatalog(
+    $card-width,
+    $card-horisontal-shift,
+    $horisontal-shift
+  );
+  padding: $vertical-shift $horisontal-shift;
+}
+
+.cards-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  background-color: $color-white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &:not(:nth-child(4n)) {
+    margin-right: $card-horisontal-shift;
+  }
+
+  &:nth-child(-n + 4) {
+    margin-bottom: $card-vertical-shift;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__image--position {
+    margin: 0 3px;
+  }
+
+  &__description {
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+    color: $color-dark;
+  }
+
+  &__description--position_top {
+    margin-top: 40px;
+  }
```

do you need `&__description--position_top`? maybe you can move it to `&__description` selector?
```suggestion
  &__description {
    line-height: 18px;
    font-size: 12px;
    font-weight: 500;
    color: $color-dark;
    margin-top: 40px;
  }
```